### PR TITLE
Removes unused size and visibility enumerations

### PR DESCRIPTION
**Pull request by dosdebug**
This PR is related to #547 which seems not a question to me but removed unused enumerations to avoid misunderstanding.

### CODE REVIEW

File: command/src/main/java/com/iluwatar/command/Wizard.java [hunk 1]
```diff
@@ -40,7 +40,9 @@
   private Deque<Command> undoStack = new LinkedList<>();
   private Deque<Command> redoStack = new LinkedList<>();
 
-  public Wizard() {}
+  public Wizard() {
```
Comment (rastdeepanshu) [inline command/src/main/java/com/iluwatar/command/Wizard.java]: Is there any use for keeping this constructor explicitly?

Comment (dosdebug) [reply command/src/main/java/com/iluwatar/command/Wizard.java]: Not really but I wanted to keep the change as small as possible.

Comment (iluwatar) [reply command/src/main/java/com/iluwatar/command/Wizard.java]: Yes, we should probably remove it.

**Comment by iluwatar** [comment]
The changes look good. Thanks @dosdebug :+1:

**Pull request merged**